### PR TITLE
Create Model parent class

### PR DESCRIPTION
**Pull request by rundef**

### CODE REVIEW

File: src/User.php [hunk 1]
```diff
@@ -8,11 +8,7 @@
  */
 namespace Corcel;
 
-use Illuminate\Database\Eloquent\Model as Eloquent;
-use Illuminate\Database\Eloquent\Relations\HasMany;
-use Illuminate\Database\Eloquent\Relations\BelongsToMany;
-
-class User extends Eloquent
+class User extends Model
```
Comment (jgrossi) [inline src/User.php]: Why did you remove `implements Authenticatable, CanResetPassword` in `User` declaration? The users' test are not passing :-(

Comment (rundef) [inline src/User.php]: My bad, I was trying to merge a branch which is out of sync with dev.

**Comment by rundef** [comment]
I'll juste make a new clean PR

**Comment by jgrossi** [comment]
Ok :-)

**Comment by rundef** [comment]
My bad I actually pushed directly in your dev branch. All the tests are passing tho. I must be too tired, time for me to get some sleep! :)

**Comment by jgrossi** [comment]
Hehehe get some rest @rundef :-)

**Comment by jgrossi** [comment]
```
PHPUnit 4.2.2 by Sebastian Bergmann.

Configuration read from /home/vagrant/Github/corcel/phpunit.xml

.................................

Time: 914 ms, Memory: 8.00Mb

OK (33 tests, 306 assertions)
```

**Comment by jgrossi** [comment]
Everything ok! Thanks @rundef :-)
